stream-video-android-ui-compose/src/main/kotlin/io/getstream/video/android/compose/ui/components/base/StreamButton.kt

```diff
@@ -165,6 +170,43 @@ public fun StreamIconButton(
     }
 }
 
+@Composable
+public fun StreamDrawableButton(
+    modifier: Modifier = Modifier,
+    @DrawableRes drawable: Int,
+    style: StreamFixedSizeButtonStyle,
+    enabled: Boolean = true,
+    interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+    onClick: () -> Unit = {},
+) {
```

Comment:
This one should be marked with the `@InternalStreamVideoApi` annotation.

Reply:
I think the intention is to let clients use these components. As I can see, none of the components from this file or the `base` package have the `@InternalStreamVideoApi` annotation. But let's ask @aleksandar-apostolov to confirm, as he originally created these base components.

Reply:
Then, we should offer the same approach. Makes sense!